santi020k/file-manager
# Feat/files

# Description

This pull request introduces several enhancements and fixes to the project. Below is a summary of the changes made:

### Fixes:
- Fix medias changes behavior.
- Fix change selected reset form.
- Add delete batch header button and other fixes.

### Features:
- Initial form logic implementation, although it requires further polishing.
- Addition of loading, private, and drive sections, along with an empty state.
- Introduction of user hooks for improved functionality.
- Upload file process refinement.
- Some adjustments to loading images for better display.

## Type of Change


## Demo

N/A

## How Has This Been Tested?

The changes have been tested thoroughly by following these steps:

1. Navigate to [specific area].
2. Perform [action] on [item].
3. Verify [result].

## Checklist

- [X] I have commented on my code, particularly in areas to understand
